src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -386,6 +439,54 @@ export default class ManualBusinessInfo extends Mixins(CountriesProvincesMixin,
     }
   }
 
+  /**
+   * Called when FileUploadPreview tells us whether a file is valid.
+   * This is called right before the File Selected event.
+   */
+  onFileValidity (valid: boolean): void {
+    this.fileValidity = valid
+  }
+
+  /**
+   * Called when FileUploadPreview tells us about a new or cleared file.
+   * This is called right after the File Validity event.
+   */
+  async onFileSelected (file: File): Promise<void> {
+    if (file) {
+      if (this.fileValidity) {
+        // try to upload to Minio
+        let psu: PresignedUrlIF
+        try {
+          psu = await this.getPresignedUrl(file.name)
+          const res = await this.uploadToUrl(psu.preSignedUrl, file, psu.key, this.getKeycloakGuid)
+          if (!res || res.status !== StatusCodes.OK) throw new Error()
+        } catch {
+          // put file uploader into manual error mode by setting custom error message
+          this.customErrorMessage = this.UPLOAD_FAILED_MESSAGE
+          this.$forceUpdate() // force file upload component to react
+          return // don't add to list
+        }
+
+        // add properties reactively to business object
+        this.$set(this.business, 'affidavitFile', {
+          name: file.name,
+          lastModified: file.lastModified,
+          size: file.size
+        } as File)
+        this.$set(this.business, 'affidavitFileKey', psu.key)
+        this.$set(this.business, 'affidavitFileName', file.name)
+        this.$set(this.business, 'affidavitFileUrl', psu.preSignedUrl)
```

Comment:
@ArwenQin , as per my recent PR, please remove this line and also line 485. We no longer save the file URL.

Reply:
Done